### ISSUE TITLE
Add builder semi-automation scripts

### ISSUE THREAD

**Issue by atolopko-czi** [chanzuckerberg/cell-census#110]
Write script to automate the following documented steps (in [Cell Census Build Process (Manual)](https://docs.google.com/document/d/12JwqpLWrm9_JbhEg99r7LB9sscevxL1gtliqu0s80Bo/)), which can be run from the builder EC2 instance:
* Cloning or updating (git pull) `cell_census` repo
* Executing build process
* Uploading census and run-time logs to S3, post-build (s3://cellxgene-data-public and s3://cellxgene-data-public-logs/builder/, respectively)
* Updating `release.json`
* Running the census_summary.py for latest & previous

Depends on: https://github.com/chanzuckerberg/cell-census/issues/109

Notebook re-running can also be automated, but we have a separate issue for that: https://github.com/chanzuckerberg/cell-census/issues/75.

**Comment by pablo-gar** [comment]
Did this get resolved here #266? @atolopko-czi

**Comment by bkmartinjr** [comment]
Closing as superseded by #265
